### DISCUSSION TITLE
FIDO U2F is not working with Google's OpenSK

### DISCUSSION THREAD

**Discussion by Fei1Yang** [dani-garcia/vaultwarden#1339] [Q&A]
### Subject of the issue
OpenSK won't be registered on the web vault.

### Your environment
* Bitwarden_rs version: 1.18.0
* Install method: built from source
* Clients used: web vault
* Reverse proxy and version: nginx/1.19.6
* Version of mysql/postgresql: mysql
* Other relevant information:
`DOMAIN` environment variable is already set.
The OpenSK security key is working with my GitHub account so I don't think it's Google's fault.
### Steps to reproduce
1. Open FIDO U2F registration page.
2. Fill the name and click "Read key".
3. Click the button on my key.

### Expected behaviour
Web vault returns the error message "There was a problem reading the security key. Try again.".

### Actual behaviour
OpenSK should be registered as a U2F security key.

### Relevant logs
```
listening for key... u2f.js:702
Extension JS API Version:  1.1 two-factor-u2f.component.ts:138
error: 5 two-factor-u2f.component.ts:138
```

**Comment by BlackDex** [comment]
I think error 5 means that either there's no https or the domain doesn't match. Please check those items first
